amida-tech/amida-notification-microservice
# Dockerfile changes per DEVOPS-365

If this PR fixes a bug, you _must_ add test cases representative of the bug.
#### What's this PR do?
Improves the Dockerfile and makes it conform to the other services' Dockerfiles per DEVOPS-365.

#### Related JIRA tickets:
https://jira.amida.com/browse/DEVOPS-365

#### How should this be manually tested?
`docker build -t SOME_TAG .`

It should build.

If you run it, it should work correctly.
